packages/fuchsia_remote_debug_protocol/pubspec.yaml

```diff
@@ -10,70 +10,14 @@ environment:
 dependencies:
   json_rpc_2: 2.2.2
   process: 4.0.0-nullsafety.4
-  web_socket_channel: 1.1.0
```

Comment:
I think you need to explicitly add stream_channel now.

Reply:
done